diagnostic.md

```diff
@@ -52,19 +67,18 @@ end
 the above `Movies` and `Profiles`.
 
 ```sh
-  # < Your Response Here >
+bundle exec rails g scaffold FavoritesFromMoviesAndProfiles
 ```
 
 1.  What is `Dependent: Destroy` and where/why would we use it?
 
 ```sh
-  # < Your Response Here >
+This line 'locks out' the ability to destroy information in a table if that information is being used by another table.  I think.
 ```
```

Comment:
Close, it essentially ensures that when we delete something, we also delete all associated objects.